aws-predictions/src/main/java/com/amplifyframework/predictions/aws/adapter/TextractResultTransformers.java

```diff
@@ -209,7 +210,13 @@ public static BoundedKeyValue fetchKeyValue(@Nullable Block block, @NonNull Map<
                 .build();
     }
 
-    private static Cell fetchTableCell(Block block, Map<String, Block> blockMap) {
+    @VisibleForTesting
```

Comment:
The `@VisibleForTesting` thing isn't ideal, and is usually used for constructor overloads.

Why not just test `fetchTable`, and keep this is a `private` implementation detail of it? If `fetchTable` passes your test cases, you don't really care about what happens inside this cell-processing routine.

Reply:
I can do that! There was no real reason for me to do it besides breaking up test suites into smaller components